shared/src/packages/orders/libs/types/order-status.type.ts

```diff
@@ -0,0 +1,7 @@
+import { type ValueOf } from '~/libs/types/value-of.type.js';
+
+import { type OrderStatus } from '../enums/order-status.enum.js';
+
+type OrderStatus = ValueOf<typeof OrderStatus>;
```

Comment:
it's better to rename it, for ex type OrderStatusValues

Reply:
done